### PR TITLE
Read method for QuestionsController

### PR DESCRIPTION
**Pull request by HollowPrincess**
https://github.com/malinink/easy-tests/issues/370
- [x] Method
- [x] Test

### CODE REVIEW

**Comment by HollowPrincess** [comment]
@malinink DOOOOONE

**Comment by HollowPrincess** [comment]
@malinink travis is OK, I don't know why coverage can't be completed for so long time, so. please mark it with a "ready for review" label

File: src/main/java/easytests/api/v1/controllers/QuestionsController.java [hunk 1]
```diff
@@ -34,11 +46,31 @@
...
      * update
      */
-    /**
-     * show(questionId)
-     */
+    @GetMapping("/{questionId}")
+    public Question show(@PathVariable Integer questionId) throws NotFoundException, ForbiddenException {
+        final QuestionModelInterface questionModel = this.getQuestionModel(
+                questionId,
+                (new QuestionsOptions()).withAnswers(new AnswersOptions())
```
Comment (malinink) [inline src/main/java/easytests/api/v1/controllers/QuestionsController.java]: I would prefere to use `this.questionsOptionsBuilder.forAuth()` and than add withAnswers.

File: src/main/resources/api/api.yaml [hunk 2]
```diff
@@ -1298,4 +1298,4 @@ definitions:
         type: integer
         format: int32
       message:
-        type: string
+        type: string
```
Comment (malinink) [inline src/main/resources/api/api.yaml]: remove spaces

File: src/test/java/easytests/api/v1/controllers/QuestionsControllerTest.java [hunk 3]
```diff
@@ -14,11 +88,79 @@
...
+        final QuestionModelInterface questionModel = new QuestionModel();
+        questionModel.map(this.questionSupport.getEntityFixtureMock(0));
+        final List<AnswerModelInterface> answersModels = new ArrayList<>();
+        IntStream.range(0, 2).forEach(idx -> {
+            final AnswerModel answerModel = new AnswerModel();
+            answerModel.map(this.answersSupport.getEntityFixtureMock(idx));
+            answersModels.add(answerModel);
+        });
+        questionModel.setAnswers(answersModels);
+        when(this.questionsOptionsBuilder.forAuth()).thenReturn(new QuestionsOptions());
```
Comment (malinink) [inline src/test/java/easytests/api/v1/controllers/QuestionsControllerTest.java]: take your attention on that mock, now it is not necessary. But when you begin use forAuth you will need to modify it. Think of "how".

**Review by malinink — CHANGES_REQUESTED** [review]

**Comment by HollowPrincess** [comment]
@malinink done. coverage can't stop again.

**Review by malinink — APPROVED** [review]

**Pull request merged**